lib/content/gittenify.js

```diff
@@ -0,0 +1,33 @@
+import catNames from 'cat-names';
+import mutationSummary from 'mutation-summary';
+import catPhoto from './catPhoto';
+import replaceUsers from './replaceUsers';
+
+export default function gittenify(store) {
+  const userMap = {};
+
+  for (const u of store.users) {
```

Comment:
@RobinThrift const should always be used when you know that the variable will not be modified. Actually the eslint config I added errors here when you use let because u never changes.